reference/commands/custom_commands/commands/create/cmd_profile.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from conan.api.conan_api import ConanAPIV2
+from conan.tools.files import save
+from conans.cli.command import conan_command, conan_subcommand
+
+profile_mac_content = """\
+[settings]
+arch=x86_64
+build_type=Release
+compiler=apple-clang
+compiler.cppstd=gnu98
+compiler.libcxx=libc++
+compiler.version=12.0
+os=Macos
+"""
+
+profile_android_content = """\
+[settings]
+os = Android
+os.api_level = 21
+arch = armv8
+
+[conf]
+tools.android:ndk_path=/usr/local/share/android-ndk
+"""
+
+
+@conan_subcommand()
+def profile_mac(conan_api: ConanAPIV2, parser, subparser, *args):
+    """
+    Create a predefined Mac OS profile file
+    """
+    subparser.add_argument("name", nargs='?',
+                           help="New name for Mac OS profile file. By default, my_mac_profile.",
+                           default="my_mac_profile")
+    args = parser.parse_args(*args)
+    profile_path = os.path.join(conan_api.cache_folder, "profiles", args.name)
+    save(None, profile_path, profile_mac_content)
```

Comment:
this ``save()`` is dirty, that is intended only for usage within ``conanfile.py``.
Here better use a python ``open().write()``